### ISSUE TITLE
minica on android device

### ISSUE THREAD

**Issue by mgip** [jsha/minica#54]
Hi guys, minica working good, but just on pc and google chrome...

Someone was able to install the Minica CA root Certificate on Android?

Mi dev domain ip is a private lan, like i said, is working on my pc but on android still show me the SSL error.

**Comment by jsha** [comment]
Yes, I've installed a minica root certificate on an Android device and used it to validate an HTTPS site. What steps have you taken so far?
